README.md

```diff
@@ -24,8 +24,8 @@ released version of Riot:
    as desired. See below for details.
 1. Enter the URL into your browser and log into Riot!
 
-Releases are signed by PGP, and can be checked against the public key
-at https://riot.im/packages/keys/riot.asc .
+Releases are signed using gpg and the OpenPGP standard, and can be checked against the public key located
+at https://packages.riot.im/keys/riot.asc .
```

Comment:
```suggestion
at https://packages.riot.im/riot-release-key.asc .
```

Reply:
I just checked on https://packages.riot.im/debian/ and the key's URL is https://packages.riot.im/debian/repo-key.asc